test/common/http/utility_test.cc

```diff
@@ -57,6 +57,15 @@ TEST(HttpUtility, isInternalRequest) {
   EXPECT_TRUE(Utility::isInternalRequest(TestHeaderMapImpl{{"x-forwarded-for", "127.0.0.1"}}));
 }
 
+TEST(HttpUtility, isWebSocketUpgradeRequest) {
+  EXPECT_FALSE(Utility::isWebSocketUpgradeRequest(TestHeaderMapImpl{}));
+  EXPECT_FALSE(Utility::isWebSocketUpgradeRequest(TestHeaderMapImpl{{"connection", "upgrade"}}));
+  EXPECT_FALSE(Utility::isWebSocketUpgradeRequest(TestHeaderMapImpl{{"upgrade", "websocket"}}));
```

Comment:
I already test for the negative case in the two lines above. Its EXPECT_FALSE. While the one below is the +ve case. Do you have something else in mind?